app/views/claim_mailer/approved.text.erb

```diff
@@ -0,0 +1,30 @@
+Dear <%= @display_name %>,
+
+We are pleased to say that your claim to get your student loan payments has been approved.
+
+We will now process your claim so we can calculate the amount of tax owed for this repayment, and then pay it for you. This will not affect the amount of money you will receive.
+
+To do this we will share your claim data with our payroll provider and HMRC. This will include:
+
+* name
```

Comment:
Should we make this "full name"? It's more accurate about the data we're sharing.